specification/domainservices/resource-manager/Microsoft.AAD/stable/2017-06-01/oucontainer.json

```diff
@@ -0,0 +1,551 @@
+          },
+          {
+            "$ref": "#/parameters/DomainServiceNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "HTTP 200 (OK) if the operation was successful.",
+            "schema": {
+              "$ref": "#/definitions/OuContainerListResult"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List of OuContainers": {
+            "$ref": "./examples/ListOuContainers.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Aad/domainServices/{domainServiceName}/ouContainer/{ouContainerName}": {
+      "get": {
+        "tags": [
+          "OuContainer"
+        ],
+        "operationId": "OuContainer_Get",
+        "summary": "Get particular OuContainer in DomainService instance",
+        "description": "Get OuContainer in DomainService instance.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DomainServiceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/OuContainerNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "HTTP 200 (OK) if the operation was successful.",
+            "schema": {
+              "$ref": "#/definitions/OuContainerOutput"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List of OuContainers": {
+            "$ref": "./examples/GetOuContainer.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "OuContainer"
+        ],
+        "operationId": "OuContainer_Create",
+        "summary": "Create OuContainer",
+        "description": "The Create OuContainer operation creates a new OuContainer under the specified Domain Service instance.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DomainServiceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/OuContainerNameParameter"
+          },
+          {
+            "name": "ouContainer",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/OuContainerInput"
+            },
+            "description": "Properties supplied to the Update OuContainer operation."
+          }
+        ],
+        "responses": {

```

Comment:
add "default" response to all your API response sections. please check other RP swaggers to get a sample. like Batch RP.

Reply:
this has been added.